From 40d65aaf14aa47975fc174c2fd12c432ac37e9af Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Mon, 30 Dec 2019 21:33:02 +0100
Subject: [PATCH 1/1] MAINTAINERS: assign drivers/rng/

With upcoming patches we will have drivers for the hardware random number
generators in drivers/rng/. As the UEFI sub-system will be the first user
of these drivers I would like to pick up the maintainership.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 MAINTAINERS | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 438fb225ab..78d4c24d33 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -781,6 +781,13 @@ F:	arch/riscv/
 F:	cmd/riscv/
 F:	tools/prelink-riscv.c
 
+RNG
+M:	Heinrich Schuchardt <xypron.glpk@gmx.de>
+S:	Maintained
+T:	git https://gitlab.denx.de/u-boot/custodians/u-boot-efi.git
+F:	drivers/rng/
+F:	drivers/virtio/virtio_rng.c
+
 ROCKUSB
 M:	Eddie Cai <eddie.cai.linux@gmail.com>
 S:	Maintained
-- 
2.24.1

